webapp/src/tutorial.tsx

```diff
@@ -432,22 +433,7 @@ export class TutorialCard extends data.Component<TutorialCardProps, TutorialCard
     }
 
     doubleClickedNextStep() {
-        const options = this.props.parent.state.tutorialOptions;
-        const { tutorialName, tutorialStepInfo, tutorialStep } = options;
-        const stepInfo = tutorialStepInfo[tutorialStep];
-        const rules = stepInfo.listOfValidationRules;
-        if (rules != undefined) {
-            const validationRuleStepStatus: pxt.Map<string | number> = {};
-            validationRuleStepStatus["tutorial"] = tutorialName;
-            validationRuleStepStatus["step"] = tutorialStep;
-            for (let i = 0; i < rules.length; i++) {
-                if (rules[i].ruleTurnOn) {
-                    validationRuleStepStatus["ruleName"] = rules[i].ruleName;
-                    let str = 'tutorial.validation.next' + (rules[i].ruleStatus ? '.pass' : '.fail');
-                    pxt.tickEvent(str, validationRuleStepStatus);
-                }
-            }
-        }
+        this.validationTelemetry('.next');
```

Comment:
small/can be done later: move the "." to be part of the helper function (`"tutorial.validation." + command`) instead of passed in